cmd/webhook/main.go

```diff
@@ -39,6 +39,7 @@ func main() {
 	injectHugepageDownApi := flag.Bool("injectHugepageDownApi", false, "Enable hugepage requests and limits into Downward API.")
 	flag.Var(&clientCAPaths, "client-ca", "File containing client CA. This flag is repeatable if more than one client CA needs to be added to server")
 	resourceNameKeys := flag.String("network-resource-name-keys", "k8s.v1.cni.cncf.io/resourceName", "comma separated resource name keys --network-resource-name-keys.")
+	resourceHonorFlag := flag.Bool("honor-resource", false, "Honor the existing resources --honor-resource")
```

Comment:
I am inclined to put the plural here. ```--honor-resources``` and also expand the description to say "Honor the existing requested resources requests & limits --honor-resources"

Reply:
sure, done